CONTRIBUTING.md

```diff
@@ -93,12 +93,13 @@ To run tests locally, please read the [CI instructions](ci/README.md).
   as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 * Make sure [CLA](https://identity.linuxfoundation.org/projects/cncf) is
   signed and CI is clear.
+* For non-trivial changes, please update the [CHANGELOG](./CHANGELOG.md).
 
 ### How to Get PRs Merged
 
 A PR is considered to be **ready to merge** when:
 
-* It has received two approvals from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
+* It has received at least one approval from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
   / [Maintainers](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer)
   (at different companies).
```

Comment:
We discussed in the SiG, and I thought we had said: "Approval from Approvers/Mainters where the approver is from a different company".   Can we update this to denote that?

Reply:
Is this already covered by line 104? But I agree we need change the word a little bit.

Reply:
Yes, it's already there. But have now reworded slightly, let me know if it still not clear.